Sync dependency versions with snews-pt.

sybenzvi opened the pull request:
PR to address #140 and SNEWS2/SNEWS_Publishing_Tools#151.

Commit:
Sync dependency versions with snews-pt.

Commit:
Renew poetry lock file; use snews-pt main branch.

Commit:
Fix obsolete reference to SNEWSMessageBuilder.

Commit:
Partially fix obsolete syntax (tests still crash).

ricmperes committed:
FIx failing CI test:
  - Change `messages.create_messages` to `messages.CoincidenceTierMessage`
  - Send messages after start listening to stream

ricmperes commented:
Test is running without failing but it is not actually waiting for the messages to show up in the stream to proceed. It should require a running instance of snews_cs on the `snews.experiments-test` topic (currently doesn't because it's not waiting to check anyway).

ricmperes committed:
Fix CI in alert - try 2:
  - Now provide messages that match the template (big ups before)
  - Change stream to be open-ended until a message comes
  - Added timeout to not be looking for messages that enver come forever

ricmperes commented:
Test now works as expected: 
  - if a coincidence server is online -> passes
  - if no coincidence server is online -> fails after 30s

⚠️ With last commit (https://github.com/SNEWS2/SNEWS_Coincidence_System/pull/142/commits/3cfd1e1cca84c987d582704e81c52cda10fdee55), the subscriber is set to run indefinitely and a timeout was set. If the timeout is not properly handled after another change, the test can be running for a long time and using up Github actions quota.

sybenzvi commented:
Note: this PR may be obsolete after the merger of PR #138.

ricmperes commented:
Superseded by #138 and #144 (updates requirements after the changes of #138)

ricmperes commented:
Above PRs now merged.
